pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -157,27 +167,27 @@ func accessTokenRunE(cmd *cobra.Command, args []string) error {
 		}
 	} else {
 		fmt.Fprintf(cmd.OutOrStdout(), "RoleBinding \"%s\" created\n", roleBinding.Name)
-		defer func() {
-			if createErr != nil {
-				err := k8sClient.Delete(context.TODO(), &roleBinding)
-				if err != nil {
-					fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete RoleBinding \"%s\", error: %s\n", roleBinding.Name, err.Error())
-				} else {
-					fmt.Fprintf(cmd.OutOrStdout(), "RoleBinding \"%s\" deleted\n", roleBinding.Name)
-				}
-			}
-		}()
+		createdRes = append(createdRes, &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "v1",
+				"kind":       "RoleBinding",
+				"metadata": map[string]interface{}{
+					"name":      roleBinding.Name,
+					"namespace": roleBinding.Namespace,
+				},
+			},
+		})
 	}
 
 	secretName := args[0]
-
 	fmt.Fprintf(cmd.OutOrStdout(), "Creating Secret \"%s\"\n", secretName)
```

Comment:
We should print the token for member to use.

Reply:
@jianjuns I feel there is no need to print the token. Member ClusterSet definition will include the secret Name information, it will be read and used by mc-controller automatically.